{{cookiecutter.project_slug}}/Makefile

```diff
@@ -60,14 +62,16 @@ setup: ### Setup remote environment
 		--name $(SETUP_NAME) \
 		--preset cpu-small \
 		--detach \
-		--volume $(PROJECT_PATH_STORAGE):$(PROJECT_PATH_ENV):ro \
+		--volume $(PROJECT_PATH_STORAGE):$(PROJECT_PATH_ENV):rw \
 		--env PLATFORMAPI_SERVICE_HOST="." \
 		$(BASE_ENV_NAME) \
 		'sleep 1h'
 	neuro cp $(REQUIREMENTS_APT) $(REQUIREMENTS_APT_STORAGE)
 	neuro cp $(REQUIREMENTS_PIP) $(REQUIREMENTS_PIP_STORAGE)
+	for file in $(PROJECT_PYTHON_FILES); do neuro cp ./$$file $(PROJECT_PATH_STORAGE)/$$file; done
 	neuro exec --no-key-check $(SETUP_NAME) "bash -c 'export DEBIAN_FRONTEND=noninteractive && $(APT_COMMAND) update && cat $(REQUIREMENTS_APT_ENV) | xargs -I % $(APT_COMMAND) install --no-install-recommends % && $(APT_COMMAND) clean && $(APT_COMMAND) autoremove && rm -rf /var/lib/apt/lists/*'"
 	neuro exec --no-key-check $(SETUP_NAME) "bash -c '$(PIP_INSTALL_COMMAND) -r $(REQUIREMENTS_PIP_ENV)'"
+	neuro exec --no-key-check $(SETUP_NAME) "bash -c '$(PIP_INSTALL_COMMAND) -e $(PROJECT_PATH_ENV)'"
```

Comment:
install user's code (for issue https://github.com/neuromation/cookiecutter-neuro-project/issues/29)

Reply:
I quite don't understand why do you need to install the package at all?
Python notebook doesn't require it.
Other tools should work fine imho.

Reply:
otherwise `from modules import foo` does not work in the jupyter notebook

Reply:
fixed not to use `setup.py`, thanks